### ISSUE TITLE
TSVs loaded with "datapusher" mode

### ISSUE THREAD

**Issue by jqnatividad** [ckan/ckanext-xloader#67]
When loading a TSV, xloader uses the slower `load_table`, datapusher style mode, not the faster `load_csv` mode.

Can't the PostgreSQL COPY command handle TSVs too?

**Comment by davidread** [comment]
Yes, sounds good. You'd just need to fiddle with COPY's DELIMITER parameter. Add a test and you're half way there.
